packages/appium/docs/en/sponsors.md

```diff
@@ -50,6 +51,11 @@ contributors!
 
 ## Bronze Sponsors
 
-<a href="https://opencollective.com/appium" target="_blank"><img src="https://opencollective.com/appium/tiers/sponsors.svg?avatarHeight=60&button=false&random=1"/></a>
+<div class="sponsor-section">
+  <a href="https://magicpod.com/" target="_blank"><img src="https://images.opencollective.com/magicpod/722b7e4/avatar/64.png" alt="MagicPod" /></a>
+  <a href="https://route4me.com/" target="_blank"><img src="../assets/images/sponsor-logo-route4me.png" alt="Route Planning and Route Mapping" /></a>
+  <a href="https://testingbot.com/" target="_blank"><img src="https://images.opencollective.com/testingbot1/aea09ee/logo/64.png" alt="TestingBot" /></a>
```

Comment:
Looks like testingbot is not in the monthly sponsor for now so we should drop this.

Hm, if we want to maintain like this individual one, we need to find a way to keep syncing active sponsors from opencollective

Reply:
yeah I guess we need to keep an eye on it manually for now. removed testingbot. seems like even using open collective's widget does not correctly show only active recurring contributors.